byos/devops-2.0/deploy/deploy.sh

```diff
@@ -30,15 +33,21 @@ if [ $# -eq 0 ]; then
 fi
 
 # Initialize parameters specified from command line
-while getopts ":l:s:" arg; do
+while getopts ":l:s:u:p:" arg; do
     case "${arg}" in
         l) # Process -l (Location)
             RESOURCEGROUPLOCATION=${OPTARG}
         ;;
         s) # Process -s (Suffix)
             RGSUFFIX=${OPTARG}
         ;;
-        \?) 
+        u) # Process -u (Username)
+            AZURE_USERNAME=${OPTARG}
+        ;;
+        p) # Process -p (Password)
+            AZURE_PASSWORD=${OPTARG} 
```

Comment:
Recommend not passing in username/password as it will be part of the bash history. Perhaps we can prompt the user to login or provide a mechanism to present SP credentials instead of a user login.

Reply:
Hey @hattan, we do not plan to commit username and password into git :)

Reply:
Hi @DariuszPorowski , sorry I should have clafiried. The concern isn't the username/password in git. But if entered when running a script then it becomes part of bash history and is available in clear text. If this isn't a concern that's ok. But I wanted to call out that it will be in bash history (not git.)

Reply:
Hi @hattan so far I commented all code related to azure login